registry/storage/driver/azure/azure.go

```diff
@@ -86,7 +86,8 @@ func New(accountName, accountKey, container, realm string) (*Driver, error) {
 	blobClient := api.GetBlobService()
 
 	// Create registry container
-	if _, err = blobClient.CreateContainerIfNotExists(container, azure.ContainerAccessTypePrivate); err != nil {
```

Comment:
Why was this `azure.ContainerAccessTypePrivate` used before and no longer used? Is it deprecated now, is it the default, or was it just not needed.